### PR TITLE
Home Stats Section 

### PR DESCRIPTION
**Pull request by Bawanthathilan**
image

### CODE REVIEW

**Review by Safnaj — CHANGES_REQUESTED** [review]
Need to increase the height of past event component card.
Other than that everything ok..!

**Comment by Bawanthathilan** [comment]
> Need to increase the height of past event component card.
> Other than that everything ok..!

please review and merge @Safnaj

File: src/app/components/member/member.component.css [hunk 1]
```diff
@@ -1,7 +1,9 @@
 #member {
   padding-bottom: 100px;
+
 }
 
+
 .member-wrapper{
   margin-left:50px;
 }
```
Comment (Shehanka) [inline src/app/components/member/member.component.css:9]: don't use px

File: src/app/components/member/member.component.css [hunk 2]
```diff
@@ -165,3 +175,49 @@ ul {
...
+.counter-box p {
+  margin: 5px 0 0;
+  padding: 0;
+  color: #909090;
+  font-size: 18px;
+  font-weight: 500
+}
+
+.counter-box i {
+  font-size: 60px;
```
Comment (Shehanka) [inline src/app/components/member/member.component.css:201]: don't use px

**Review by Shehanka — CHANGES_REQUESTED** [review]

**Pull request merged**